omicron_server/__main__.py

```diff
@@ -50,7 +61,23 @@ def create_root_user(session):
         session.add(user)
 
 if __name__ == "__main__":
+    arguments = docopt(__doc__)
```

Comment:
From what I understand, the ```docopt``` constructor [requires a string](https://docopt.readthedocs.org/en/0.2.0/#api) to build the options parser. However, it doesn't necessarily need to be a [module docstring](https://www.python.org/dev/peps/pep-0257/). I'm not sure if we want to move the docopt stuff to a separate string right under the module docstring. This would let us write a better module description for the server docs.